File: Sorting algorithms/bubble.py
def bubblesort(arr):
  l = len(arr)
  for _ in range(l):
    for i in range(l-1):
      if arr[i]>arr[i+1]: arr[i],arr[i+1]=arr[i+1],arr[i]
  return arr

arr=[5,3,7,9,3,6,67,23,12,6,421,1,0]
print(bubblesort(arr))

File: Sorting algorithms/bubble.js
function bubble(arr){
  for(i=0;i<=arr.length-1;i++){
    for (j=0;j<=arr.length-1;j++){
      if (arr[j]>arr[j+1]){
        [[arr[j]],[arr[j+1]]]=[[arr[j+1]],[arr[j]]];
      }
      else{
        continue;
      }
    }
  }
  return arr;
}
  
var arr = [456,23,888,123,34,33,1,0,45,99,320,50];
var l = 0;
var h = arr.length-1;
console.log("Here is unsort array: "+arr);
console.log("Here is sorted array: "+bubble(arr,l,h));
